tampermonkey_scripts/README.md

```diff
@@ -9,3 +9,12 @@ the `Holdings` tab of the brokerage account that you want to copy from. When the
 the button will copy the holdings into JSON data to your clipboard which can then be pasted into
 the GoogleSheets or any other program that wants to use the JSON data.
 
+## etoro_holdings_getter.js
+
+This script will run on etoro.com/portfolio and place a grey `Copy Holdings` button next to navbar buttons. 
```

Comment:
I wonder if maybe a screenshot of the location where the button would be helpful? I'm just wondering since for the other scripts I have the video tutorials, I might actuallay also add an image of the screenshoot from personal capital. What do you think? 

I have some subscribers that are getting some interest in this script 👍